lib/livebook/runtime/definitions.ex

```diff
@@ -183,6 +183,21 @@ defmodule Livebook.Runtime.Definitions do
           packages: [kino_explorer]
         }
       ]
+    },
+    %{
+      kind: "Elixir.Kino.RemoteCell",
+      name: "Remote cell",
+      requirement_presets: [
+        %{
+          name: "Default",
+          packages: [
+            %{
+              name: "kino",
+              dependency: %{dep: {:kino, github: "livebook-dev/kino", branch: "main"}, config: []}
+            }
+          ]
```

Comment:
```suggestion
          packages: [kino]
```

You can change https://github.com/livebook-dev/livebook/blob/836ebbc81917868a5f80efc3e6a0c19c1d62d97e/lib/livebook/runtime/definitions.ex#L6 to `{:kino, github: "livebook-dev/kino", override: true}`